### PR TITLE
search-app: fix display condition

### PR DESCRIPTION
**Pull request by kpsherva**

### CODE REVIEW

File: invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/Results.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const Results = ({ currentResultsState = {} }) => {
     useContext(SearchConfigurationContext);
   const multipleLayouts = layoutOptions.listView && layoutOptions.gridView;
   return (
-    (total) && (
+    (total || null) && (
```
Comment (ntarocco) [inline invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/Results.js:31]:
```suggestion
    total === 0 && (
```

**Review by ntarocco — APPROVED** [review]
LGTM! Added a minor suggestion :)

File: invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/Results.js (hunk 1 above)
Comment (kpsherva) [reply invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/Results.js:31]: unfortunately still displaying zero if implemented this way

**Comment by ntarocco** [comment]
I am sorry if I insist, but this has to be improved. It is not clear why the `|| null`, and it will lead to another bug in the feature when someone will change it (if I see that, I will for sure change it :) ).

**Pull request merged**